packages/core/src/types/runtime.ts

```diff
@@ -90,23 +90,36 @@ export interface TableDataSourceMeta {
 }
 
 export type DataSourceMeta = TableDataSourceMeta | ImageDataSourceMeta;
-export interface DataAccessor<T = DefaultType> {
+export interface SequentialDataAccessor<T = DefaultType> {
   next: () => Promise<Sample<T> | null>;
   nextBatch: (batchSize: number) => Promise<Array<Sample<T>> | null>;
   seek: (pos: number) => Promise<void>;
 }
+export interface DataAccessor<T = DefaultType> extends SequentialDataAccessor {
+  init: (size: number) => void;
```

Comment:
`init` should not be a part of interface `DataAccessor`, because script will never use it.

Reply:
okey, remove `init` function definition from DataAccessor.